doc/nestml_language.md

```diff
@@ -459,7 +463,7 @@ These functions can be used to implement custom buffer lookup logic but should b
 
 ## Equations
 
-### Synaptic response
+### Shape functions
```

Comment:
Details required regarding the dimension of shape objects (later modification to NESTML code are ambiguous)